### PR TITLE
Fixes #21446 - adjust to rubocop v0.51.0

### PR DESCRIPTION
**Pull request by swapab**

### CODE REVIEW

File: lib/foreman_maintain/reporter/cli_reporter.rb [hunk 1]
```diff
@@ -1,4 +1,3 @@
-require 'thread'
```
Comment (iNecas) [inline lib/foreman_maintain/reporter/cli_reporter.rb]: Still required for ruby 1.8.7

Comment (swapab) [reply lib/foreman_maintain/reporter/cli_reporter.rb]: Disabled `Lint/UnneededRequireStatement:` in `.rubocop.yml`

Comment (iNecas) [reply lib/foreman_maintain/reporter/cli_reporter.rb]: What about disabling it just here, with a comment on why?

Comment (swapab) [reply lib/foreman_maintain/reporter/cli_reporter.rb]: Yeah, I tried that actually but `rubocop` is complaining about it saying `Unnecessary disabling of Lint/UnneededRequireStatement`

```
Offenses:
lib/foreman_maintain/reporter/cli_reporter.rb:1:18: W: Unnecessary disabling of Lint/UnneededRequireStatement.
require 'thread' # rubocop:disable Lint/UnneededRequireStatement
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
157 files inspected, 1 offense detected
RuboCop failed!
```

https://travis-ci.org/theforeman/foreman_maintain/jobs/293000847
https://travis-ci.org/theforeman/foreman_maintain/jobs/293001743

Comment (swapab) [reply lib/foreman_maintain/reporter/cli_reporter.rb]: Have reported this to https://github.com/bbatsov/rubocop/issues/4940

**Comment by iNecas** [comment]
Ok, int that case, let's disable it globally for now, until the issue gets resolved in rubocop

**Comment by iNecas** [comment]
Thanks @swapab

**Pull request merged**